lib/collection/src/wal_delta.rs

```diff
@@ -24,11 +28,14 @@ pub struct RecoverableWal {
     /// This means two things:
     /// - this WAL has at least all these clock versions (same as `highest_clocks`)
     /// - this WAL cannot resolve any delta below any of these clocks
+    //
+    // TODO: Rename to `oldest_available_records` or `oldest_resolvable_records`?
+    // TODO: What does "same as `highest_clocks`" note above mean?
```

Comment:
It means that entries in `cutoff_clocks` are never newer than `highest_clocks`.

So, if we update cutoff, we also update `highest_clocks`.

Reply:
> It means that entries in cutoff_clocks are never newer than highest_clocks.
> So, if we update cutoff, we also update highest_clocks.

I do get this... but I still don't see how "same as `highest_clocks`" can mean this. I'll just reword the quote above and replace this part later. :D

Reply:
It notes 'at least', but I agree it is confusing.

> I'll just reword the quote above and replace this part later. :D

Thanks!